applications/luci-app-upnp/luasrc/controller/upnp.lua

```diff
@@ -19,8 +19,12 @@ function index()
 end
 
 function act_status()
+	local uci = luci.model.uci.cursor()
+	local lease_file = uci:get("upnpd", "config", "upnp_lease_file")
+	
 	local ipt = io.popen("iptables --line-numbers -t nat -xnvL MINIUPNPD 2>/dev/null")
 	if ipt then
+		local upnpf = io.open(lease_file, "r")
```

Comment:
Please use `local upnpf = lease_file and io.open(lease_file, "r")` in order to avoid a runtime error if the lease file happens to be unset.